os/mm/mm_heap/mm_partition_mgr.c

```diff
@@ -83,13 +83,15 @@ void mm_initialize_ram_partitions(void)
 	uint32_t start;
 	uint32_t size;
 
-	start = (uint32_t)__ksram_segment_start__ + (uint32_t)__ksram_segment_size__;
-	size = (REGION_SIZE - (uint32_t)__ksram_segment_size__) / CONFIG_NUM_APPS;
+	start = (uint32_t)__usram_segment_start__ + (uint32_t)__usram_segment_size__;
+	size = (REGION_END - start) / CONFIG_NUM_APPS;
 	for (i = 0; i < CONFIG_NUM_APPS; i++) {
 		g_mempartitions[i].start = ALIGN_EVEN(start);
 		g_mempartitions[i].size = size;
 		g_mempartitions[i].status = MM_PART_FREE;
 		start += size;
+
+		mvdbg("App RAM partition_%d start = 0x%x size = %d\n", i, g_mempartitions[i].start, g_mempartitions[i].size);
```

Comment:
Shouldn't it be size = %u ?  size is uint32_t ...

Reply:
Done